documentation/src/docs/beginner_dev_guide.md

```diff
@@ -13,7 +17,7 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-# Pravega - Creating Your First Application
+import versions from '@site/versions';
```

Comment:
The right-side index for this page seems incomplete (it starts by point 4). Also, the section titles look excessively large, as main title all the time.

Reply:
fixed title sizes and right-side index